bydbctl/internal/cmd/root.go

```diff
@@ -152,3 +163,10 @@ func bindTLSRelatedFlag(commands ...*cobra.Command) {
 		c.Flags().StringVarP(&cert, "cert", "", "", "Certificate for tls")
 	}
 }
+
+func bindUsernameAndPasswordFlag(commands ...*cobra.Command) {
```

Comment:
To bind them in RootCmdFlags, all sub commands will inherit them.By that, the cmd pkg will only change root.go.